### PR TITLE
feat: FeedbackButton (DX-2405)

### PR DESCRIPTION
**Pull request by mikaalnaik**
**Fixes or implements VF-XXX**

### Related PRs

- https://github.com/voiceflow/XXXXXXXXX/pull/123

### Checklist

- [ ] Breaking changes have been communicated, including:
    - New required environment variables
    - Renaming of interfaces (API routes, request/response interface, etc)
- [ ] New environment variables have [been deployed](https://www.notion.so/voiceflow/Add-Environment-Variables-be1b0136479f45f1adece7995a7adbfb)
- [ ] Appropriate tests have been written
    - Bug fixes are accompanied by an updated or new test
    - New features are accompanied by a new test

### CODE REVIEW

File: packages/chat/src/components/ColorTheme/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { ColorTheme } from './ColorTheme.component';
```
Comment (gillyb) [inline packages/chat/src/components/ColorTheme/index.ts:1]: we don't need this file..  
(I saw that on the ui library there is an index.ts file in each dir, but it's not needed. we can just export what we need in the files themselves. It only makes sense to have them when exporting specifically for another library to use - like in our case, the 'ui.ts' or 'stories.ts' files)

File: packages/chat/src/components/FeedbackButton/FeedbackButton.test.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import { FeedbackButton } from '.';
+
+describe('Button', () => {
+  it('onClick callback fires properly', async () => {
+    const onClick = vi.fn();
+    const MOCK_CONFIG = { render: { mode: 'embedded' }, verify: { projectID: ' ' } } as any;
+
+    render(
+      <RuntimeProvider config={MOCK_CONFIG} assistant={{ persistence: {}, extensions: [], color: 'blue' } as any}>
```
Comment (gillyb) [inline packages/chat/src/components/FeedbackButton/FeedbackButton.test.tsx:14]: can be removed i think..

**Review by gillyb — APPROVED** [review]
minor comments left

File: packages/chat/src/components/ColorTheme/ColorTheme.component.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { createPalette } from '@/styles/colors';
+
+import { container, derivativeColor, derivativeColorContainer, primaryColorBlock } from './ColorTheme.css';
+
+export const ColorTheme = ({ color }: { color: string }) => {
+  const bezier = createPalette(color);
```
Comment (ninabondar) [inline packages/chat/src/components/ColorTheme/ColorTheme.component.tsx:6]: why is it called bezier here 😅

File: packages/chat/src/__fixtures__/colors.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export const COLOR_FIXTURE = [
+  'orange',
+  '#397DFF',
+  '#A3E4D7',
+  'red',
+  'orange',
```
Comment (ninabondar) [inline packages/chat/src/__fixtures__/colors.ts:6]: two oranges

File: packages/chat/src/components/FeedbackButton/FeedbackButton.css.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+          },
+        },
+      },
+    },
+  },
+});
+
+export const iconStyle = recipe({
+  base: {
+    color: '#656D75',
```
Comment (ninabondar) [inline packages/chat/src/components/FeedbackButton/FeedbackButton.css.ts:47]: is this desired here? (no token)

**Review by ninabondar — APPROVED** [review]

**Pull request merged**